### PR TITLE
add sub0 banner

### PR DESCRIPTION
**Pull request by carinaluise**
In this PR:
The decoded banner has been updated to sub0
& a tiny footer fix that i spotted from last time

### CODE REVIEW

File: kusama-guide/docusaurus.config.js [hunk 1]
```diff
@@ -171,7 +171,7 @@ module.exports = {
     announcementBar: {
       id: "banner",
       content:
-        'Join Polkadot Decoded 2023 on June 28-29! <a href="https://polkadot.network/ecosystem/events/decoded-2023/?utm_source=guide.kusama.network&utm_medium=referral&utm_campaign=decoded23&utm_content=notification&utm_term=two" target="_blank" rel="noopener noreferrer">Register now 👉</a>',
+        'sub0 is back! Lisbon, Portugal, Sep 19-20. <a href="https://polkadot.network/ecosystem/events/sub0/?utm_source=guide.kusama.network&utm_medium=referral&utm_campaign=sub023&utm_content=notification" target="_blank" rel="noopener noreferrer">Learn more 👉</a>',
```
Comment (DrW3RK) [inline kusama-guide/docusaurus.config.js:174]:
```suggestion
        'Sub0 is back! Lisbon, Portugal, Sep 19-20. <a href="https://polkadot.network/ecosystem/events/sub0/?utm_source=guide.kusama.network&utm_medium=referral&utm_campaign=sub023&utm_content=notification" target="_blank" rel="noopener noreferrer">Learn more 👉</a>',
```

File: polkadot-wiki/docusaurus.config.js [hunk 2]
```diff
@@ -287,7 +287,7 @@ module.exports = {
     announcementBar: {
       id: "banner",
       content:
-        'Join Polkadot Decoded 2023 on June 28-29! <a href="https://polkadot.network/ecosystem/events/decoded-2023/?utm_source=wiki.polkadot.network&utm_medium=referral&utm_campaign=decoded23&utm_content=notification&utm_term=two" target="_blank" rel="noopener noreferrer">Register now 👉</a>',
+        'sub0 is back! Lisbon, Portugal, Sep 19-20. <a href="https://polkadot.network/ecosystem/events/sub0/?utm_source=wiki.polkadot.network&utm_medium=referral&utm_campaign=sub023&utm_content=notification" target="_blank" rel="noopener noreferrer">Learn more 👉</a>',
```
Comment (DrW3RK) [inline polkadot-wiki/docusaurus.config.js:290]:
```suggestion
        'Sub0 is back! Lisbon, Portugal, Sep 19-20. <a href="https://polkadot.network/ecosystem/events/sub0/?utm_source=wiki.polkadot.network&utm_medium=referral&utm_campaign=sub023&utm_content=notification" target="_blank" rel="noopener noreferrer">Learn more 👉</a>',
```

**Review by DrW3RK — COMMENTED** [review]
Just a small nit. Was lowercase intentional?

File: kusama-guide/docusaurus.config.js (hunk 1 above)
Comment (filippoweb3) [inline kusama-guide/docusaurus.config.js:174]: @DrW3RK I think sub0 is lowercase, at least on the official website everything is lowercase. 

My other question is: **Why not promoting ParisDOT.comm that comes first?**

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**